__tests__/postService.test.js

```diff
@@ -1,23 +1,62 @@
 const { createPost } = require("../src/services/postService");
 const { sendPost } = require("../src/repository/postDAO");
+const postDAO = require("../src/repository/postDAO");
+const uuid = require('uuid');
+
+jest.mock('../src/repository/postDAO');
+const mockDatabase = [];
```

Comment:
You use an array here, but use `Map` methods like `set` and `clear` later. Either make this a map or replace the map methods with array methods.